packages/api/src/routes/medical/patient.ts

```diff
@@ -326,6 +328,7 @@ const medicalRecordFormatSchema = z.enum(mrFormat);
  *        Accepts "pdf", "html", and "json". If provided, the Webhook payload will contain a signed URL to download
  *        the file, which is active for 3 minutes. If not provided, will send json payload in the webhook.
  * @param req.body Optional metadata to be sent through Webhook.
+ * @param req.aiBrief Optional flag to include an AI-generated medical record brief into the medical record summary.
```

Comment:
Since we're leaving the check and throwing error if this is set but the feature is not enabled for the cx, let's add a note here indicating it must be enabled on account level before this can be used?